src/adapters/listing-wizard/v1.ts

```diff
@@ -241,19 +296,45 @@ export class ListingWizardAdapterV1 extends Adapter {
     return this.contract.DOMAIN_SEPARATOR();
   }
 
-  private prepareListingParams(assetListingParams: AssetListingParams): {
-    encodedAssets: Assets.AssetStruct[];
-    listingParams: Listings.ParamsStruct;
-    maxLockPeriod: BigNumberish;
-    immediatePayout: boolean;
-  } {
-    const { assets, params, maxLockPeriod, immediatePayout } = assetListingParams;
-    const encodedAssets = assets.map(x => this.encodeAsset(x));
-    const listingParams: Listings.ParamsStruct = {
-      lister: this.accountIdToAddress(params.lister),
-      configurator: this.accountIdToAddress(params.configurator),
-    };
+  private async createBatch(listings: CreateListingParams[], buffer: CreateListingParams[]): Promise<string[]> {
```

Comment:
Can you explain what this `buffer` is for?

Reply:
I think I will rename it - it is the original listings array (putting back objects to it when batch is over the limit)

Reply:
@syakunin will also add some sleep calls in the loops (to avoid node rate limits)

Reply:
> I think I will rename it - it is the original listings array (putting back objects to it when batch is over the limit)

Why not just return it?

Reply:
fixed